src/components/common/rest_api.ts

```diff
@@ -115,3 +115,23 @@ function toFormData(file: File) {
   formData.append('upload', file);
   return formData;
 }
+
+function isCloudWorkstation(url: string) {
+  return url.includes('cloudworkstations.dev');
```

Comment:
let's prepend with a "\." so we don't accidentally catch other domains that just happen to end in this string, see @maneesht comment in the other implementation of this